specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/ManagedPrivateEndpoints_Create.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "factoryName": "exampleFactoryName",
+    "managedVirtualNetworkName": "exampleManagedVirtualNetworkName",
+    "managedPrivateEndpointName": "exampleManagedPrivateEndpointName",
+    "ifMatch": null,
+    "managedPrivateEndpoint": {
+      "properties": {
+        "description": "A managed private endpoint",
```

Comment:
>description [](start = 9, length = 11)

Why is description not being returned in PUT response?

Reply:
Removed the description. It is not supported. Thanks for catching the issue